Integrations/integration-Moloch.yml

```diff
@@ -128,8 +132,13 @@ script:
             }
 
             if (jsonResponses.indexOf(command) !== -1) {
-                if (command == 'moloch_sessions_json') {
-                    return parseSessions(JSON.parse(res.Body));
+                if (command == 'moloch_sessions_json' && res.Body) {
+                    var respBody = JSON.parse(res.Body);
+                    if (respBody && respBody.data) {
+                        return parseSessions(respBody);
+                    } else {
+                        return respBody;
+                    }
                 } else {
                     return JSON.parse(res.Body);
```

Comment:
`return 'No sessions data was found'` otherwise we would get exception for empty response